### DISCUSSION TITLE
How to apply global-styles.css (with Tailwind) to global.error.tsx file?

### DISCUSSION THREAD

**Discussion by maloyuso** [vercel/next.js#61164] [App Router] [answered]
I'm including global-error.tsx boundary on my NextJS 14.1.0 app router application.
When I manually trigger an error, the styles and fonts are not being applied to that element.

I'm already adding the same configuration as the root layout. (html/head/body tags, global styles import, font classNames, etc)

**Comment by ananuness** [comment, marked as answer]
Hi, Yuso!! 

As the docs states, we have to add the `global-styles.css` to *RootLayout* for our app to have access to tailwind styles, and considering that `global-error.tsx` is at the same level as the root layout, you will have to import `global-styles.css` in this file too!

**Comment by sanderquirynen** [comment]
Sorry to bump this old discussion. But when I do this it doesn't seem to work.
I see the stylesheet being loaded in the network tab, but it isn't being added to the <head> section for some reason.

Next.js version 14.2.23

**Reply by soohoonc** [reply]
Having the same issue as well on Next.js 14.2.13

Imported stylesheet, see it in the network tab, but it just isn't able to recognize the class names? perhaps something to do with tailwind?

**Reply by sanderquirynen** [reply]
I downgraded to Next.js 14.1.0 and it's fixed. So I don't think it's a Tailwind issue.
If I find the time I'll upgrade step by step to see in which version it breaks.

**Reply by sergio-milu** [reply]
happening to me 14.2.15

**Reply by nathanielmoon** [reply]
Experiencing this with `"next": "14.2.8"`

**Reply by tmsick** [reply]
same in 14.2.17

**Reply by sommeeeer** [reply]
same here at `14.2.24`

**Reply by forjut** [reply]
same in 14.2.18

**Reply by sommeeeer** [reply]
got it to work on `14.2.24` by using `import './(main)/globals.css';`

here is my full code `global-error.tsx` for reference : https://gist.github.com/sommeeeer/7a01701cc3b5781ff8fa231b9fcf5703

to throw an error you can use this: https://gist.github.com/sommeeeer/0fc8ce7e5fffaf4a1e6f125168f12b65

remember its only possible to trigger the `global-error.tsx` in `next start` or once deployed on that version of Next 14
